src/mame/taito/taito_h.cpp

```diff
@@ -52,6 +52,7 @@ Known issues :
  - Sprite zoom is a bit wrong.
  - Title screen of dynamite league is wrong a bit, which is mentioned by
    Yasuhiro Ogawa.
+ - Flip screen doesn't work fine on background layers
```

Comment:
What do you mean by “doesn’t work fine”?  It doesn’t work?  It doesn’t work at a fine (as opposed to coarse) level?

Reply:
You know perfectly well the users first language is not English hence sometimes there can be translation issues
dont be so elitest, it's attitudes like that which turn people away from wanting to helpout this project

Reply:
There is nothing elitist about asking for some more clarification when something is not clear.

Reply:
Well it's clearly a typo or a translation issue i suppose it should have read as such.......

- Flip screen doesn't work on background layers

There are ways to ask for clarification which dont come across as being possibly condesending

Reply:
It could mean:
* The layout of background tilemaps is flipped (works at a “coarse”) level but the tiles themselves are not flipped (doesn’t work at a “fine” level).
* It “works” but doesn’t “work fine” – the background layers are flipped but there’s some other issue (e.g. layer alignment).
* Background layers aren’t flipped at all.

I *don’t* know what it means – I haven’t tried running these games in cocktail cabinet mode.

It’s better to make the comments explicit and unambiguous, i.e. to describe the incorrect behaviour, both to help people unfamiliar with the system and to make it easier to tell whether the bug was fixed and the comment wasn’t updated or it’s a subtle bug you aren’t noticing immediately.

Reply:
Updated notes to add more details  on flip screen issue for each game in the driver